src/components/NavBar/index.js

```diff
@@ -3,14 +3,17 @@ import { Layout, Menu } from 'antd';
 import { Link } from 'react-router-dom';
 import PropTypes from 'prop-types';
 
-const { Header } = Layout;
+const { Header, Sider } = Layout;
+// Window width of 500px
+const mobileWidthBreak = 500;
```

Comment:
500 seems bit of random. Let' use grid system. https://ant.design/components/grid/